keys/employees-keys/README.md

```diff
@@ -3,172 +3,117 @@
+* [FAQ](#faq)
+  - [What to do when this key expires?](#what-to-do-when-this-key-expires)
+  - [If we extend expiry every time, why not set "key does not
+    expire"?](#if-we-extend-expiry-every-time-why-not-set-key-does-not-expire)
+  - [What to do when private key was lost or
+    compromised?](#what-to-do-when-private-key was-lost-or-compromised)
+
 ## Adding key to the repository
 
 To add new key to this repository you should:
 
 1. [Create new GPG key](#how-to-create-a-new-gpg-key)
-2. [Send key for signing](#sending-key-for-signing)
+2. [Encrypt revocation cert](#encrypt-revocation-cert)
+3. [Send key for signing and revocation
+   cert](#sending-key-for-signing-and-revocation-cert)
 
 Then 3mdeb Team Leader or Management should
 
-1. [3mdeb Team Leader/Management: Identity confirmation](#identity-confirmation)
+1. [3mdeb Team Leader/Management: Identity
+   confirmation](#identity-confirmation)
 2. [3mdeb Team Leader/Management: Key signing](#key-signing)
 
 Finally you should configure your git and provide signed key through pull
 request:
 
 1. [Git configuration](#git-configuration)
 2. [Create Pull Request with signed key](#create-a-pull-request-with-key)
+3. [Upload to keys.opengpg.org](#upload-to-keys-openpgp-org)
 
 If you have questions, please check [FAQ](#FAQ) section. Maybe the answer is
 already there. If not, feel free to submit an issue to the repository.
 
-### How to create a new GPG key
+### Create a new GPG key
 
 The following assumes that you have `gpg` installed.  The interaction is broken
 into sections that correspond to user queries.
 
-```
-$ gpg --gen-key
-```
-
-For GPG version 2, you should use `gpg --full-generate-key`. Please note that
-depending on the version of GPG you use, your output may look different.
-
-This is what you'll see if `gpg` is configured for the first time:
-
-```
-gpg (GnuPG) 2.2.27; Copyright (C) 2021 Free Software Foundation, Inc.
-This is free software: you are free to change and redistribute it.
-There is NO WARRANTY, to the extent permitted by law.
-```
-
-Pick "RSA and RSA" key:
-
-```
-Please select what kind of key you want:
-   (1) RSA and RSA (default)
-   (2) DSA and Elgamal
-   (3) DSA (sign only)
-   (4) RSA (sign only)
-Your selection? 1
-```
-
-Use 4096 bits for key size:
+Clone repository:
 
-```
-RSA keys may be between 1024 and 4096 bits long.
-@What keysize do you want? (2048) 4096
-Requested keysize is 4096 bits
+```shell
+git clone https://github.com/3mdeb/3mdeb-secpack.git
 ```
 
-Use a 2 years expiry key and then confirm your choice:
+Change directory:
 
-```
-Please specify how long the key should be valid.
-         0 = key does not expire
-      <n>  = key expires in n days
-      <n>w = key expires in n weeks
-      <n>m = key expires in n months
-      <n>y = key expires in n years
-Key is valid for? (0) 2y
-Key expires at Fri 07 Apr 2023 11:40:33 AM CEST
-Is this correct? (y/N) y
+```shell
+cd 3mdeb-secpack/scripts
```

Comment:
Do we have to execute it from `scripts`, not the root dir?

In a typical flow, we first use this script, and then move to the next section. We would need to go back one directory level so commands from the [Encrypt revocation cert](https://github.com/3mdeb/3mdeb-secpack/blob/5b47704c3380fd7f773dc425114ba4ae9f7cba4c/keys/employees-keys/README.md#encrypt-revocation-cert) work fine

Reply:
@macpijan, from testing the most recent version, it doesn't matter which location we run the script. The directory should be writable by user-run script since we create temporary files. Do you expect me to change anything here?

Reply:
@macpijan ping?